extension/awsxrayproxy/extension.go

```diff
@@ -19,29 +19,39 @@ import (
 
 	"go.opentelemetry.io/collector/component"
 	"go.uber.org/zap"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/proxy"
 )
 
 type xrayProxy struct {
 	logger *zap.Logger
 	config *Config
+	server proxy.Server
 }
 
 var _ component.Extension = (*xrayProxy)(nil)
 
 func (x xrayProxy) Start(ctx context.Context, host component.Host) error {
-	// TODO(anuraaga): Implement me
+	go x.server.ListenAndServe()
+	x.logger.Info("X-Ray proxy server started")
```

Comment:
Perhaps a bit more parity on logging?

https://github.com/aws/aws-xray-daemon/blob/90a5344676b88c1a867ceee3cbce33b07600a946/pkg/proxy/server.go#L133-L136

Or is this taken care of by extension interface?